scripts/dist.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# exit on error
+set -e
+# echo commands
+set -x
+
+VERSION=$(grep '"version"' package.json | sed 's/.*"version": *"\([^"]*\)".*/\1/')
+TAR_FLAGS="--no-acls"
+
+# prevent ._ files from being included in the archive on macOS
+[ "$(uname -s)" = "Darwin" ] && TAR_FLAGS+=" --no-mac-metadata"
```

Comment:
This is better but still not entirely portable. There was [GNU-Darwin](https://en.wikipedia.org/wiki/Darwin_(operating_system)) and others that will match this but may not support `--no-mac-metadata`, which is a non-standard option that only exists on `bsdtar`, but doesn't exist on GNU tar or OpenBSD tar.

The suggested cpio option, `find dist -not -name '._*' | cpio -H ustar -o | gzip -9 > gi-v${VERSION}.debug.tgz`, should work everywhere.

Reply:
No, it doesn't work because `._` files don't exist on macOS so the `find` command won't work

Reply:
That `find` command should still work whether the `._` files exist or not.